### ISSUE TITLE
Enemy count

### ISSUE THREAD

**Issue by DrHardSlump** [nicholasclark-artist/DCG#97]
**Arma 3 Version:** `Last` (stable / rc / dev)
**CBA Version:** `Last`
**DCG Version:** `3.2`

**Server Mods:**
- `@CBA_A3`
- `@dcg`
- `@rhs`
- `@ace`

**Description:**
- Change the enemy count on occupy locations (you cut this from serverconfig in last update)

**Comment by nicholasclark-artist** [comment]
This setting got lost in the *dcg_occupy* redesign. The unit count in occupied locations already increases based on player count, but I can add a multiplier.

**Comment by DrHardSlump** [comment]
Yes plz... A multiplier will be good, need more people to kill on our server.

Thx"!!

**Comment by DrHardSlump** [comment]
Can the multiplier also affect the missions?

**Comment by nicholasclark-artist** [comment]
What do you mean affect the mission? This multiplier would apply to any unit created by the occupy addon.
